src/modules/decklink/consumer_decklink.cpp

```diff
@@ -539,7 +539,8 @@ class DeckLinkConsumer
 
 		if ( !m_sliced_swab && mlt_properties_get( consumer_properties, "sliced_swab" )
 			&& mlt_properties_get_int( consumer_properties, "sliced_swab" ) )
-			m_sliced_swab = mlt_slices_init(0, SCHED_FIFO, sched_get_priority_max( SCHED_FIFO ) );
+			m_sliced_swab = mlt_slices_init_pool(0, SCHED_FIFO,
+				sched_get_priority_max( SCHED_FIFO ), __FILE__ );
```

Comment:
I have separate pool because of each service require different priority. decklink callback called from IRQ and required RT performance. On other hand using global pool could postpone some task executing...